sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -522,17 +517,7 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
 
       npmResolutions := Map.empty,
 
-      additionalNpmConfig := Map.empty,
-
-      //          Webpack.libraries(
-      //            generatedWebpackConfigFile,
-      //            customWebpackConfigFile,
-      //            webpackResourceFiles,
-      //            entryPointFiles,
-      //            entries,
-      //            targetDir,
-      //            log
-      //          ).to[Set]
+      additionalNpmConfig := Map("private" -> JSON.bool(true)),
```

Comment:
Agreed. It's extracted now, see https://github.com/scalacenter/scalajs-bundler/pull/161